src/components/input/input-base.ts

```diff
@@ -110,12 +108,13 @@ export abstract class IgcInputBaseComponent
 
   protected abstract renderInput(): TemplateResult;
 
+  //Remove from here since we wont have value prop in input-base
   protected resolvePartNames(base: string) {
     return {
       [base]: true,
       prefixed: this.prefixes.length > 0,
       suffixed: this.suffixes.length > 0,
-      filled: !!this.value,
+      //filled: !!this.value,
```

Comment:
I don't see this added back in extending classes though, I'm assuming its added for a reason, right?